[WIP] Analyze Section Updates (#211)
HumanCellAtlas/data-portal-content: content/analyze/visualization-components/visualization-components.md

    - ./igv.md
---

## Visualization Packages

> The HCA Visualization Package Registry lists JavaScript modules that enable analysis through interactive data exploration on the web.  

Reviewer: ```suggestion
Visualization packages listed below are JavaScript modules that enable analysis through interactive data exploration on the web.  
```

>  These solutions are not built by the HCA DCP implementation team but are solutions we foster and support. 

Reviewer: A bit shorter and less redundant, while preserving Tim's original intent to distinguish:

```suggestion
These solutions are built by third parties, and fostered by the HCA DCP.
```

> Portal developers can find instructions for integrating these visualizations into a web app on each component's detail page. Sample code showing a minimal reference implementation is available for some components by clicking on their "mashup" badge.

Reviewer: Let's delete or (preferably, if possible) comment out this part about detail pages until they're ready.

```suggestion
```

   

>#### Submit Your Component

> >If you are a visualization engineer and would like to add your software to this list, [please submit your component](https://github.com/HumanCellAtlas/data-portal-content/issues/new/?template=submit-visualization-component-package.md).

Reviewer: ```suggestion
>[Submit your visualization](https://github.com/HumanCellAtlas/data-portal-content/issues/new/?template=submit-visualization-component-package.md)
```
